training/tests/integration/aicon/test_cicd_aicon_04_icon-dream_medium.yaml

```diff
@@ -250,33 +251,36 @@ training:
     iterations: 300000
     min: 3.0e-07
     warmup: 1000
-  variable_loss_scaling:
-    default: 1
-    pl:
-      P: 1
-    sfc: {}
+  scalers:
+    general_variable:
+      _target_: anemoi.training.losses.scalers.GeneralVariableLossScaler
+      weights:
+        default: 1
+        P: 1
+    pressure_level:
+      _target_: anemoi.training.losses.scalers.ReluVariableLevelScaler
+      group: pl
+      y_intercept: 0.2
+      slope: 0.001
+    node_weights:
+      _target_: anemoi.training.losses.scalers.GraphNodeAttributeScaler
+      nodes_name: ${graph.data}
+      nodes_attribute_name: area_weight
+      norm: unit-sum
   metrics: []
+  variable_groups:
+    default: sfc
+    pl: [q, t, u, v, w, z]
```

Comment:
Dear @JPXKQX et al., thank you very much for updating the AICON test case.

I have a question for clarification: Is the `ReluVariableLevelScaler` actually used for any variable in this version of the configuration? I'm just wondering as the variables in this list are not part of the dataset used in this test.

Reply:
Hi, I think you're right and at the moment if q, t, u, v, w and z are not in your dataset, then this pressure level scaler is simply not being applied. Would it be correct to change this pl group just to P?

Reply:
Yes, I think we could even use all atmospheric variables, i.e.  `P, T, U, V`. And would it be possible to rename the group to `ml`, as the ICON data is based on model levels?